src/core/device_model.jl

```diff
@@ -32,15 +32,15 @@ thermal_gens = DeviceModel(ThermalStandard, ThermalBasicUnitCommitment),
 mutable struct DeviceModel{D <: PSY.Device, B <: AbstractDeviceFormulation}
     feedforward::Union{Nothing, AbstractAffectFeedForward}
     use_slacks::Bool
-    duals::Vector{<:ConstraintType}
+    duals::Vector
```

Comment:
Just curious, what is the reason for this change? Are we no longer bounding the types?

Reply:
This is actually a `Vector{DataType}`.

Reply:
So is it supposed to be defined as `Vector{DataType}`?

Reply:
yes, @sourabhdalvi we can make this `Vector{DataType}` since it stores types of constraints.